validator/accounts/v2/wallet.go

```diff
@@ -129,6 +186,45 @@ func (w *Wallet) WriteKeymanagerConfigToDisk(ctx context.Context, encoded []byte
 	return nil
 }
 
+// Writes the password file for an account namespace in the wallet's passwords directory.
+func (w *Wallet) writePasswordToFile(accountName string, password string) error {
+	passwordFilePath := path.Join(w.passwordsDir, accountName+passwordFileSuffix)
+	passwordFile, err := os.OpenFile(passwordFilePath, os.O_CREATE|os.O_RDWR, os.ModePerm)
+	if err != nil {
+		return errors.Wrapf(err, "could not create password file in directory: %s", w.passwordsDir)
+	}
+	defer func() {
+		if err := passwordFile.Close(); err != nil {
+			log.Errorf("Could not close password file: %s", passwordFilePath)
+		}
+	}()
+	n, err := passwordFile.WriteString(password)
+	if err != nil {
+		return errors.Wrap(err, "could not write account password to disk")
+	}
+	if n != len(password) {
+		return fmt.Errorf("could only write %d/%d password bytes to disk", n, len(password))
+	}
+	return nil
+}
+
+// Generates a human-readable name for an account. Checks for uniqueness in the accounts path.
+func (w *Wallet) generateAccountName() (string, error) {
+	var accountExists bool
+	var accountName string
+	for !accountExists {
+		accountName = petname.Generate(numAccountWords, "-" /* separator */)
```

Comment:
Seems like this generates the same series of account names every time...will have to confirm its randomness

Reply:
Fixed